pkg/tfgen/docs.go

```diff
@@ -2183,39 +2201,67 @@ func (c infoContext) fixupPropertyReference(text string) string {
 			// This is a resource name
 			resname, mod := resourceName(c.info.GetResourcePrefix(), name, resInfo, false)
 			modname := formatModulePrefix(parentModuleName(mod))
-			switch c.language {
-			case Golang, Python:
-				// Use `ec2.Instance` format
-				return open + modname + resname.String() + close
-			default:
-				// Use `aws.ec2.Instance` format
-				return open + c.pkg.String() + "." + modname + resname.String() + close
+
+			// Use `ec2.Instance` format for Go and Python
+			goAndPyFormat := open + modname + resname.String() + close
+			// Use `aws.ec2.Instance` format for all other languages
+			allOtherLangs := open + c.pkg.String() + "." + modname + resname.String() + close
```

Comment:
So the text we get is already appropriately snake/pascal-cased, we don't need to handle that?

Reply:
Yeah! The resource name is already Pulumified.